cartography/intel/aws/ec2.py

```diff
@@ -847,6 +888,50 @@ def load_ec2_vpcs(neo4j_session, data, region, current_aws_account_id, aws_updat
         )
 
 
+TAG_RESOURCE_TYPE_MAPPINGS = {
+    'instance': {'label': 'EC2Instance', 'property': 'instanceid'},
+    'network-interface': {'label': 'NetworkInterface', 'property': 'id'},
+    'security-group': {'label': 'EC2SecurityGroup', 'property': 'id'},
+    'subnet': {'label': 'EC2Subnet', 'property': 'subnetid'},
+    'vpc': {'label': 'AWSVPC', 'property': 'id'},
+}
+
+TAG_INGEST_TEMPLATE = Template("""
+MATCH (n:$label{$property: {id}})
+WITH n
+MERGE (tag:Tag{key: {Key}})
```

Comment:
Let's add another label here: `:Tag` can be a general label, and for AWS assets we can call this an `AWSTag` or something similar.

We would also have to add the generic `:Tag` label [to GCPNetworkTags](https://github.com/lyft/cartography/blob/1148c98e58238a83833a4a20f620fa49df1bf009/cartography/intel/gcp/compute.py#L542).